medusa/post_processor.py

```diff
@@ -129,32 +122,28 @@ def _checkForExistingFile(self, existing_file):
             EXISTS_SAME if the file exists and is the same size as the file we are post processing
         """
         if not existing_file:
-            self._log(u"There is no existing file so there's no worries about replacing it", logger.DEBUG)
+            logger.debug(u"There is no existing file so there's no worries about replacing it")
             return PostProcessor.DOESNT_EXIST
 
         # if the new file exists, return the appropriate code depending on the size
         if os.path.isfile(existing_file):
-
             # see if it's bigger than our old file
             if os.path.getsize(existing_file) > os.path.getsize(self.file_path):
-                self._log(u'File {0} is larger than {1}'.format(existing_file, self.file_path), logger.DEBUG)
+                logger.debug(u'File {0} is larger than {1}'.format(existing_file, self.file_path))
```

Comment:
You can use lazy logging: without formatting the msg:

`logger.debug('My msg {abc}',  abc='def')`